src/components/ContractInfo.tsx

```diff
@@ -15,15 +18,47 @@ const InfoItem = ({
 );
 
 const ContractInfo = () => {
-  const { GiveStableFee, WantStableFee } =
+  const { GiveStableFee, WantStableFee, MintLimit } =
```

Comment:
while here, please correct the param names so they're consistent
(s/Stable/Minted)
https://github.com/Agoric/dapp-psm/blob/f772ba79db7e0d4230b52d55ca6eb566f94a8d3c/src/utils/updates.ts#L26-L28

Reply:
also consider dropping the capitalization to match JS conventions of `StudlyCase` for types and `camelCase` for values.